src/containers/Header/Header.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './Header.scss';
+import { Link } from 'react-router-dom';
+import logo from '../../assets/lienflash-logo.png'
+
+// need to figure out conditional logic to display buttons and adjust logo when on different views
```

Comment:
I've got some media queries setup for the form that we can use to adjust the logo size.

Reply:
I talked to my mentor about the Header and she suggested that we could always have two versions of it and render the correct Header version on each of the components. This could be easier than conditionally rendering certain buttons on it when on certain pages, but I'm not sure if there would be any disadvantages to this approach, something to think about and we can discuss potential pros/cons.